.dprint.jsonc

```diff
@@ -4,6 +4,7 @@
     "newLineKind": "auto",

```

Comment:
I could change this but that feels like a bit much as it'll redo quite a few things, actually. Maybe that's okay and I should just override this in general?